lib/realtime/postgres_cdc.ex

```diff
@@ -60,23 +60,35 @@ defmodule Realtime.PostgresCdc do
     end
   end
 
-  @spec aws_to_fly(String.t()) :: nil | <<_::24>>
-  def aws_to_fly(aws_region) when is_binary(aws_region) do
-    case aws_region do
-      "us-east-1" -> "iad"
-      "us-west-1" -> "sea"
-      "sa-east-1" -> "iad"
-      "ca-central-1" -> "iad"
-      "ap-southeast-1" -> "syd"
-      "ap-northeast-1" -> "syd"
-      "ap-northeast-2" -> "syd"
-      "ap-southeast-2" -> "syd"
-      "ap-south-1" -> "syd"
-      "eu-west-1" -> "lhr"
-      "eu-west-2" -> "lhr"
-      "eu-west-3" -> "lhr"
-      "eu-central-1" -> "lhr"
-      _ -> nil
+  @doc """
+  Translates a region from a platform to another
+
+  If the platform is `:fly` then the region is translated from an AWS region to a Fly region
+  If the platform is `:aws` then the region is not translated
+  """
+  @spec platform_region_translator(String.t()) :: nil | binary()
+  def platform_region_translator(aws_region) when is_binary(aws_region) do
+    platform = Application.get_env(:realtime, :platform)
+
+    if platform == :fly do
+      case aws_region do
+        "us-east-1" -> "iad"
+        "us-west-1" -> "sea"
+        "sa-east-1" -> "iad"
+        "ca-central-1" -> "iad"
+        "ap-southeast-1" -> "syd"
+        "ap-northeast-1" -> "syd"
+        "ap-northeast-2" -> "syd"
+        "ap-southeast-2" -> "syd"
+        "ap-south-1" -> "syd"
+        "eu-west-1" -> "lhr"
+        "eu-west-2" -> "lhr"
+        "eu-west-3" -> "lhr"
+        "eu-central-1" -> "lhr"
+        _ -> nil
+      end
+    else
+      aws_region
```

Comment:
Do we want to have RT nodes in all Supabase regions?

Reply:
doubt it and we could a check just to validate that we're in an allowed region

something along the lines of 
```elixir
aws_region in ["us-east-1", "eu-west-1", ...]
```

Reply:
`aws_region` is the region where the tenant's db resides. This function is responsible for initiating a database connection in the closest region. For example, if a tenant's database is located in aws's `us-east-1`, then we should establish a connection on a node in fly's `iad` region.
```elixir
"us-east-1" -> "iad"
```

Reply:
Yeah and you're right for some reason I had the assumption that we were deploying in every Supabase tenant region and use said RDS conn.

Will check what can be done